plugins/directpv.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: directpv
+spec:
+  version: v2.0.1
+  homepage: https://github.com/minio/directpv
+  shortDescription: CSI driver to manage drives in k8s cluster as volumes
```

Comment:
Some issues here:

1. This description is for the driver, not the plugin. We should maybe rewrite that?
2. Words like k8s are redundant here because users are already in a k8s context.
3. it would be better to properly capitalize concept/API names here (e.g. volume → Volume) and below.

Reply:
@ahmetb  :addressed the comments. PTAL